src/hera/workflows/v5/examples/upstream/steps.py

```diff
@@ -15,14 +15,14 @@
     generate_name="steps-",
     entrypoint="hello-hello-hello",
 ) as w:
-    with Steps(name="hello-hello-hello"):
+    with Steps(name="hello-hello-hello") as s:
         Step(
             name="hello1",
             template="whalesay",
             arguments=[Parameter(name="message", value="hello1")],
         )
 
-        with ParallelSteps():
+        with s.parallel():
```

Comment:
Nice addition 💯 Makes it (more) obvious it's part of `s`.

Extending on this change - should we make `Parallel` a private class so it's not directly instantiable? i.e. what if I have `with Parallel` directly under a `with Workflow`?